publish-maven.gradle

```diff
@@ -14,6 +14,11 @@ install {
 
 def customizePom(pom, gradleProject) {
 	pom.whenConfigured { generatedPom ->
+
+		generatedPom.dependencies = generatedPom.dependencies.findAll {
+			dep -> dep.artifactId != "jackson-bom"
```

Comment:
Do we still have a proper BOM import in the generated pom?

Reply:
So, is it expected to have it like this in the final pom:
```
  <dependency>
      <groupId>com.fasterxml.jackson.core</groupId>
      <artifactId>jackson-core</artifactId>
      <scope>compile</scope>
      <optional>true</optional>
    </dependency>
    <dependency>
      <groupId>com.fasterxml.jackson.core</groupId>
      <artifactId>jackson-databind</artifactId>
      <scope>compile</scope>
      <optional>true</optional>
    </dependency>
```
and don't have any `dependencyManagement` block in the pom?

I see SF uses `id "io.spring.dependency-management" version "1.0.7.RELEASE" apply false` instead of `platform` dependency.
Although I believe that we can live with built-in Gradle solution, only the problem is to fix a `publish-maven.gradle` script properly to resolve actual versions for those dependencies.
Unfortunately we just can't have a  `dependencyManagement` in the library's pom, I guess...

Reply:
Hmmm - good catch - I assumed that would just happen automatically; we could make jackson the parent like I did i the SI samples that use jackson.

If you don't like that solution (I am not keen on it), then, yes, more work is needed. If you have any ideas how to do it, please take a look when you have time, my gradle/groovy skills are lacking.

Reply:
As a quick solution, I suggest to forget a standard Gradle approach and do whatever it is in SF with that `io.spring.dependency-management` plugin. Let's hope it still works well with the latest Gradle version!